src/Sfx/App/Scripts/Models/DataModels/Cluster.ts

```diff
@@ -30,6 +32,29 @@ module Sfx {
             super(data);
         }
 
+        public checkExpiredCertStatus() {
+            try {
+                if (!ClusterHealth.certExpirationChecked) {
+                //Check cluster health
+                //if healthy then no cert issue
+                //if warning/Error
+                    //starting walking and query all seed nodes in warning state for cluster cert issues
```

Comment:
We only care about seed nodes?

Reply:
talking with Dragos, in a concern to not potentially ping every node in the cluster for health events we decided that just pinging seed nodes should provide a high confidence of one of those nodes providing info of an expired cert.

Reply:
Sounds good, thanks!